src/test/kotlin/com/codingchallenge/transactionanalyser/TransactionLoaderTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.codingchallenge.transactionanalyser
+
+import com.codingchallenge.transactionanalyser.domain.TransactionType
+import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Test
+import java.math.BigDecimal
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+internal class TransactionLoaderTest{
+    private val transactionLoader = TransactionLoader(
+        this::class.java.classLoader.getResourceAsStream("transactions.csv") ?: throw IllegalArgumentException("File not found")
+    )
+    private val formatter = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss")
+
+    @Test
+    fun givenCsv_WhenLoadTransactionsFromCsv_ThenTheExpectedTransactionsReturned(){
```

Comment:
it is generally good practice in kotlin tests to use the special Junit feature of writing the function name without camel case and use backticks. For example, `fun givenCsv_WhenLoadTransactions...`  can turn into `fun `givenCsv_WhenLoadTransactions``

Reply:
sorry I can't get the formatting right. What I mean is like this https://stackoverflow.com/a/65923081/11281137

Reply:
Edit: that is a kotlin feature not a Junit feature sorry